node_operations.asciidoc

```diff
@@ -518,7 +518,7 @@ To secure your operating system, here are some of the top items to consider:
 . Maintenance - Make sure that you keep your operating system up to date. Enable automated daily or weekly installation of security updates.
 . Least Privilege - Set up users for specific processes and give them the least access needed to run a service. Do not run processes with admin privileges (e.g. root).
 . Process Isolation - Use the operating system features to isolate processes from each other.
-. File System Permissions - Configure the file system carefully, on the least-privilege principle. Do not make files readable or writeable by everyone.
+. File System Permissions - Configure the file system carefully, on the least-privilege principle. Do not make files readable or writable by everyone.
```

Comment:
is this correct?

Reply:
See my other comment, 
Also search the chmod man page for "writable" you can find it there. 
https://www.man7.org/linux/man-pages/man1/chmod.1.html